style.css

```diff
@@ -510,8 +1014,556 @@ body {
     height: 100%;
     -o-object-fit: cover;
     object-fit: cover;
-    -o-object-position: 0 -10px;
-    object-position: 0 -10px;
+    -o-object-position: 0 0;
+    object-position: 0 0;
+  }
+
+  .middle-section__image {
+    height: 264px;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    overflow: hidden;
+    margin-top: 0;
+    padding: 0;
+  }
+
+  .middle-section__image {
+    height: 264px;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    overflow: hidden;
+    margin-top: 20px;
+    padding: 0;
+  }
+
+  .middle-section__link {
+    display: none;
+  }
+}
+
+@media (min-width: 1200px) {
+  .middle-section__link {
+    position: absolute;
+    left: 60%;
+    top: 90%;
+    cursor: pointer;
+    text-decoration: none;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    color: #000;
+  }
+}
+
+@media (min-width: 992px) and (max-width: 1200px) {
+    .middle-section__link {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)